go-controller/pkg/informer/informer_test.go

```diff
@@ -138,19 +137,19 @@ var _ = Describe("Informer Event Handler Tests", func() {
 		deletes := int32(0)
 
 		k := fake.NewSimpleClientset(
-			&v1.Namespace{
+			&kapi.Namespace{
 				ObjectMeta: metav1.ObjectMeta{
 					UID:  types.UID(namespace),
 					Name: namespace,
 				},
-				Spec:   v1.NamespaceSpec{},
-				Status: v1.NamespaceStatus{},
+				Spec:   kapi.NamespaceSpec{},
+				Status: kapi.NamespaceStatus{},
 			},
 		)
 
 		f := informers.NewSharedInformerFactory(k, 0)
 
-		e := NewDefaultEventHandler(
+		e, err := NewDefaultEventHandler(
```

Comment:
make sure err is nil with a expect gomega test
`Expect(err).NotTo(HaveOccurred())`